### PR TITLE
Relayer: comment review

### PR DESCRIPTION
**Pull request by EndymionJkb**

### CODE REVIEW

File: pkg/standalone-utils/contracts/relayer/AaveWrapping.sol [hunk 1]
```diff
@@ -47,13 +47,14 @@ abstract contract AaveWrapping is IBaseRelayerLibrary {
...
         // The wrap caller is the implicit sender of tokens, so if the goal is for the tokens
-        // to be sourced from outside the relayer, we must first them pull them here.
+        // to be sourced from outside the relayer, we must first pull them here.
         if (sender != address(this)) {
             require(sender == msg.sender, "Incorrect sender");
             _pullToken(sender, dynamicToken, amount);
         }
 
         dynamicToken.approve(address(staticToken), amount);
+        // Use 0 for the referral code
```
Comment (TomAFrench) [inline pkg/standalone-utils/contracts/relayer/AaveWrapping.sol:57]: Do we want to set one of these up maybe?

**Review by TomAFrench — APPROVED** [review]
Looks good!

File: pkg/standalone-utils/contracts/relayer/AaveWrapping.sol (hunk 1 above)
Comment (EndymionJkb) [reply pkg/standalone-utils/contracts/relayer/AaveWrapping.sol:57]: I was wondering about that. Is it worth passing one in at construction time? I could add that

File: pkg/standalone-utils/contracts/interfaces/IStaticATokenLM.sol [hunk 2]
```diff
@@ -22,8 +22,8 @@ interface IStaticATokenLM is IERC20 {
      *   0 if the action is executed directly by the user, without any middle-man
      * @param fromUnderlying bool
      * - `true` if the msg.sender comes with underlying tokens (e.g. USDC)
-     * - `false` if the msg.sender comes already with aTokens (e.g. aUSDC)
-     * @return uint256 The amount of StaticAToken minted, static balance
+     * - `false` if the msg.sender already has aTokens (e.g. aUSDC)
```
[the remark this answers is not in the document]
Comment (TomAFrench) [reply pkg/standalone-utils/contracts/interfaces/IStaticATokenLM.sol:25]: Alternatively we could ask if Aave is interested in these updates seeing as they haven't deployed yet?

[the remark this answers is not in the document]
Comment (nventuro) [reply pkg/standalone-utils/contracts/interfaces/IStaticATokenLM.sol:25]: Ah that'd be good, yes.

File: pkg/standalone-utils/contracts/relayer/AaveWrapping.sol (hunk 1 above)
Comment (nventuro) [reply pkg/standalone-utils/contracts/relayer/AaveWrapping.sol:57]: Aave has confirmed that they are not and will not use these referral codes, so we don't need to do anything.

**Pull request merged**